.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    name: Build and Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version: [14, 16]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v3
+      with:
+        node_version: ${{ matrix.node_version }}
+
+    - name: Prepare
+      run: |
+        npm install --global yarn
+        yarn install
+
+    - name: Build
+      run: yarn run build
+
+    - name: Test
+      run: yarn run test
+
+  deploy:
```

Comment:
Can we please remove these automatic release jobs?

For now, we will not do this, and we would need to check on how to properly implement this first since we also merge unreleased features to master.